packages/syft/tests/conftest.py

```diff
@@ -204,3 +208,9 @@ def mongo_client(testrun_uid):
     "tests.syft.action_graph.fixtures",
     "tests.syft.serde.fixtures",
 ]
+# patching compute heavy hashing to speed up tests
+SyftClientSessionCache._get_key = (
+    lambda email, password, connection: f"{email}{password}{connection}"
+)
+user.salt_and_hash_password = cache(user.salt_and_hash_password)
+user.check_pwd = cache(user.check_pwd)
```

Comment:
This should be provisioned through `monkeypatch.setattr` in the fixture instead of making it a global patch?

Reply:
Good point! I applied the changes.